bears/c_languages/ClangBear.py

```diff
@@ -52,17 +52,18 @@ def diff_from_clang_fixit(fixit, file):
     return Diff.from_string_arrays(file, new_file)
 
 
-def sourcerange_from_clang_range(range):
+
```

Comment:
this extra blank line needs to be removed per PEP8 standard

Reply:
done